google-cloud-storage/samples/acceptance/buckets_test.rb

```diff
@@ -581,4 +582,38 @@
       bucket.public_access_prevention = :inherited
     end
   end
+
+  describe "storage move object" do
+    let(:file_1_name) { "file_1_name_#{SecureRandom.hex}.txt" }
+    let(:file_2_name) { "file_2_name_#{SecureRandom.hex}.txt" }
+    let :hns_bucket do
+      hierarchical_namespace = Google::Apis::StorageV1::Bucket::HierarchicalNamespace.new enabled: true
+      storage_client.create_bucket random_bucket_name do |b|
+        b.uniform_bucket_level_access = true
+        b.hierarchical_namespace = hierarchical_namespace
+      end
+    end
+    let :create_file_hns do
+      file_content = "A" * (3 * 1024 * 1024) # 3 MB of 'A' characters
+      file = StringIO.new file_content
+      hns_bucket.create_file file, file_1_name
+    end
+    it "obejct is moved and old object is deleted" do
```

Comment:
```suggestion
    it "obejct is moved and old object is deleted" do
```
it "object is moved and old object is deleted" do